src/convertToObject.js

```diff
@@ -6,7 +6,35 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // This onbject will be returned.
+  const styledObject = {};
+
+  // Who the hell write sourceString like that? My grandma can do better...
+  // If i ever see this during actuall job, i'll leave the company.
+  //                                         (through window)
+  const cleanString = sourceString
+    .replace(/\s*;\s*$/, '')
+    .replace(/\s*:\s*/g, ':')
+    .trim();
```

Comment:
[CODE KNOWLEDGE] This series of replace methods can be a bit confusing. It might be easier to understand if you split the string directly into properties and values, and then trim the whitespace from each of them individually.